### ISSUE TITLE
Performance implications of user testing with mechamarkers

### ISSUE THREAD

**Issue by zepumph** [phetsims/ratio-and-proportion#28]
@BLFiedler brought up today that mechamarkers was taking 40% of his CPU. This felt like much more than is needed. @Petroochio and @clementzheng do you have any idea why this may be happening? Is it possible to turn off specific modules of that to improve performance.

This is worrisome given the whole user stack on potentially not-so-great computers. 
* The sim in a browser
* Zoom
* Any assistive technology that would accompany access to the sim
 * Mechamarkers

@BLFiedler and I found that the CPU percentage dropped 10% just by toggling video in the mechamarkers app. That may be helpful to recommend during recordings.

Is there anything else we can do to optimize this performance?

**Comment by BLFiedler** [comment]
![image](https://user-images.githubusercontent.com/47331395/80024549-77ece580-849c-11ea-8b27-e1428ea5db1a.png)

![image](https://user-images.githubusercontent.com/47331395/80024686-b1255580-849c-11ea-8662-0b03bad11614.png)

References, if at all useful.

**Comment by BLFiedler** [comment]
Zoom, shared screen of sim with mechamarkers app and Toggle Video On, just to show a typical use scenario

![image](https://user-images.githubusercontent.com/47331395/80025171-707a0c00-849d-11ea-835c-a1940d9f44ae.png)

**Comment by zepumph** [comment]
I want to add that my mechamarkers runs around 20% CPU and doesn't drop noticeably when turning off my video.

**Comment by zepumph** [comment]
Over in #89, it looks like tangible implementation is not going to be included in the initial publication of this simulation. Unassigning for now.

**Comment by Petroochio** [comment]
Hey @zepumph! Sorry I missed the notification for this before. Yeah, we've been aware of the memory issues, but presently we are working on a browser only version that should be less intensive. In addition, I'm switching some of the image processing to the GPU which should cut down on CPU use. I'll let you know as soon as it's stable.

**Comment by zepumph** [comment]
That's awesome! Thanks for the heads up. Over in https://github.com/phetsims/ratio-and-proportion/issues/89 we discussed the tangible implementation timeline, and it looks like we are at least 6 months out. Let's touch base the next time we are developing with mechamarkers.

**Comment by BLFiedler** [comment]
Reviving this as a reminder to speak with ATLAS folk to see the latest state of mechamarkers.

**Comment by zepumph** [comment]
I would say we shouldn't spend any time on this issue until we integrate their newer, in-browser solution and test it out.

**Comment by BLFiedler** [comment]
Details here are being discussed in other opened issues.
